docs/user-guide/flow-computation/continuous-aggregation.md

```diff
@@ -182,7 +188,7 @@ The above query puts the data from the `ngx_access_log` table into the `ngx_coun
 It calculates the distinct country for each time window.
 The `date_bin` function is used to group the data into one-hour intervals.
 The `ngx_country` table will be continuously updated with the aggregated data,
-providing real-time insights into the distinct countries that are accessing the system. 
+providing real-time insights into the distinct countries that are accessing the system. The `EXPIRE AFTER` make flow ignore data with `access_time` older than 7 days and no longer update them, see more explain in [manage-flow](manage-flow.md#expire-after).
```

Comment:
What does the `no longer update them` mean?
The `access_time` column is in the source table, which never be updated by the flow.
I guess what you mean is that Flow will not use expired data for aggregation calculations. Perhaps you could say "don't calculate them anymore"?